voxblox/include/voxblox/simulation/objects.h

```diff
@@ -142,7 +139,7 @@ class Cube : public Object {
 
   virtual bool getRayIntersection(const Point& ray_origin,
                                   const Point& ray_direction,
-                                  FloatingPoint max_dist,
+                                  FloatingPoint /*max_dist*/,
```

Comment:
@helenol  this could be a bug, in the other objects you used this max_dist parameter, but here it's not used. wdyt?

Reply:
Yeah this is a bug. Should add one last check for the distance before returning. THANKS!!!!!!
```
     if (t > max_dist) {
       return false;
     }
```